src/main/java/mate/academy/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.service;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private static MovieDao movieDao;
```

Comment:
```suggestion
    private MovieDao movieDao;
```
no need to make it static